CodeEdit/Features/Editor/Views/EditorView.swift

```diff
@@ -68,7 +86,14 @@ struct EditorView: View {
         }
         .focused($focus, equals: editor)
         .onReceive(NotificationCenter.default.publisher(for: NSNotification.Name("CodeEditor.didBeginEditing"))) { _ in
-            editor.temporaryTab = nil
+            if navigationStyle == .openInTabs {
+                editor.temporaryTab = nil
+            }
+        }
+        .onChange(of: navigationStyle) { newValue in
+            if newValue == .openInPlace && editor.tabs.count == 1 {
+                editor.temporaryTab = editor.tabs[0]
```

Comment:
I'm not sure I like making it a temporary tab by default. Thinking of this UX scenario:
- Open file in place
- Make edits
- Need another tab
- Switch to open in tabs
- Open file - old file replaced and have to go find that old file.

Reply:
This was patterned after Xcode. Command click will eventually open a new tab when in the open in place navigation style so this doesn't happen. I will open an issue for this after we get this merged.